provider/src/main/java/org/sunbird/aws/snsclient/SnsClientFactory.java

```diff
@@ -0,0 +1,27 @@
+package org.sunbird.aws.snsclient;
+
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.regions.Region;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.sns.AmazonSNSClient;
+import com.amazonaws.services.sns.model.SetSMSAttributesRequest;
+
+/**
+ * Created by nickpack on 09/08/2017.
+ */
+public class SnsClientFactory {
```

Comment:
@shriharshs I suggest that this factory implements a consistent interface which can be implemented by other factories which provide SMS sending channels.

cc:@coolbung